index.html

```diff
@@ -1,10 +1,43 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <style>
+    @import url('https://fonts.googleapis.com/css2?family=Doto:wght@700&family=Lato&display=swap');
```

Comment:
https://stackoverflow.com/questions/1022695/difference-between-import-and-link-in-css